Final step

muja-b opened the pull request.

muja-b committed:
modified the add to handle any amount of numbers

muja-b committed:
step3&4

added the option to choose a delimiter
and handled the new line case

muja-b committed:
designed an exception for negatives numbers

added a function to check for negative numbers and to throw an exception if it encountered one

muja-b committed:
added a function to filter out numbers larger then 1000

sudqijawabreh commented:
didn't we have two projects one for the code and one for the tests?  Where is the project for the code ?

muja-b commented:
its in the using.cs 
vs code wont detect the code otherwise

muja-b committed:
added step#6

modified the existing tests and added the option for the delimiter to be a string

sudqijawabreh commented on Kata/Calculator.Tests/Usings.cs:
```diff
@@ -13,12 +13,18 @@ class mainClass
-        return number1+number2;
-     }   
+        var myint=Parse(mynums);
+        var mysum=myint.Aggregate((sum,x)=>sum+=x);
```
linq .Sum

sudqijawabreh commented on Kata/Calculator.Tests/Usings.cs:
```diff
@@ -13,12 +13,18 @@ class mainClass
     {
      public int add(string numbers){
-        if(numbers.Length==0)return 0;
+        if(numbers.Equals(""))return 0;
```
.IsNullOrEmpty

sudqijawabreh commented on Kata/Calculator.Tests/Usings.cs:
```diff
@@ -22,10 +22,22 @@ class calculator
         return mysum;
      }
 
+        private negitaveList checkNegitaveNumbers(List<int> myint)
```
Get list of negative numbers could done the same

muja-b committed:
fixed step #6

in the case of multi letter delimiter the delimiter is now surrounded with brackets

muja-b committed:
hotfix

fixed the get delimiter function

muja-b committed:
supported the multi delimiters

converted the delimiters to  a list
and handled the use of it
